config.json

```
@@ -423,6 +423,19 @@
                 "structs"
             ]
         },
+        {
+            "slug": "resistor-color-trio",
+            "uuid": "39e831d0-ea7a-4df9-b9cb-64ccb996ee0e",
+            "core": false,
+            "unlocked_by": "resistor-color-duo",
```

```suggestion
            "unlocked_by": null,
```

`resistor-color-duo` hasn't been implemented on this track yet, so this won't work. We can leave it as a floating exercise that the student can do at any time for now until I figure out where it fits in with the progression. Feel free to change it to a different exercise if you want, though.